### DISCUSSION TITLE
How add static and dynamic frameworks in iOS ?

### DISCUSSION THREAD

**Discussion by pulnec** [callstack/react-native-builder-bob#290] [Q&A]
I need add two types framework in the podspec file for use in the library.
the dynamic framework add without problem but the static framework show the error **"No such module"**

**Comment by gokul-kulkarni** [comment]
@pulnec did u find a way out?
I am having somewhat similar issue.
The native SDK that I wanna use is exposed as swift package.
It gives me "No such module" error.

**Comment by pulnec** [comment]
not yet, I don't understand why it happens your sdk has a static type framework? sentro example has a .swiftmodule file and a .a file?

**Comment by yachaka** [comment]
@pulnec I am having the same issue. Did you find a way ?

**Comment by nazmeln** [comment]
Any updates on this one? 😅
@pulnec Did you solve it somehow?

**Comment by Meyanis95** [comment]
Hey, 

I have the opposite problem. 
I have 2 xcframework, one is static and the other one is dynamic.

No issues if I place the static one into `vendored_frameworks`: 
```
s.vendored_frameworks = 'ios/Frameworks/MyStaticSDK.xcframework'
```

But I have the error below when I add the dynamic lib the same way. 
```
s.vendored_frameworks = 'ios/Frameworks/*.{xcframework}'
```

`root/ios/Frameworks:`
```
MyStaticSDK.xcframework
MyDynamicSDK.xcframework
```

```
Installing my-sdk-react-native 0.1.0
Generating Pods project
[!] Unable to install vendored xcframework `MyDynamicSDK` for Pod `my-sdk-react-native` because it contains dynamic libraries which are not supported.
Use dynamic frameworks for dynamic linking instead.
```

So I'm curious about how you added the dynamic framework to the RN pod?

**Reply by wcandillon** [reply]
Were you able to fix this issue? I am encountering it as well.

**Reply by Meyanis95** [reply]
Unfortunately not

**Reply by objectiveSee** [reply]
@wcandillon did you ever resolve this?

**Reply by wcandillon** [reply]
no never we ended up using static libs
